blueprints/ember-cli-bootstrap-bookingsync-sass/index.js

```diff
@@ -16,6 +16,6 @@ module.exports = {
   },
 
   afterInstall() {
-    return this.addPackageToProject('sweetalert2');
+    return this.addPackageToProject('sweetalert2@7.3.5');
```

Comment:
I think this is not necessary since it will pick the specified version in the `package.json`. I see you've already locked in the version there.

Reply:
I've tried it locally and without version declared here it will import latest version.

Reply:
When you tried it locally did you use your local branch of the addon or the remote one?

Reply:
remote one.